 2018-12-06 14:02:58,043 [P9144/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.12.4 application starting on ARIEL-41
 2018-12-06 14:02:58,291 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2018-12-06 14:02:58,327 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 36ms)
 2018-12-06 14:02:58,557 [P9144/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2018-12-06 14:02:58,570 [P9144/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2018-12-06 14:02:58,574 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:02:59,580 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1005ms)
 2018-12-06 14:02:59,581 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2018-12-06 14:02:59,586 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 5ms)
 2018-12-06 14:02:59,703 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:02:59,704 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2018-12-06 14:02:59,704 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2018-12-06 14:02:59,708 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 3ms)
 2018-12-06 14:02:59,709 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:02:59,709 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:02:59,710 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2018-12-06 14:02:59,715 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 5ms)
 2018-12-06 14:02:59,753 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:02:59,754 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:02:59,754 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2018-12-06 14:02:59,759 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 4ms)
 2018-12-06 14:02:59,759 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:02:59,759 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:02:59,759 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2018-12-06 14:02:59,764 [P9144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 4ms)
 2018-12-06 14:03:19,291 [P9144/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: ARIEL-42\MSSQLSERVER14, InitialCatalog: Roznamah2. Exception: System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:-1,State:0,Class:20
 2018-12-06 14:03:19,292 [P9144/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-12-06 14:03:19,808 [P9144/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: ARIEL-42\MSSQLSERVER14, InitialCatalog: Roznamah2. Exception: System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:-1,State:0,Class:20
 2018-12-06 14:03:19,808 [P9144/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-12-06 14:03:20,809 [P9144/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: ARIEL-42\MSSQLSERVER14, InitialCatalog: Roznamah2. Exception: System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:-1,State:0,Class:20
 2018-12-06 14:03:20,810 [P9144/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-12-06 14:03:21,811 [P9144/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: ARIEL-42\MSSQLSERVER14, InitialCatalog: Roznamah2. Exception: System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:-1,State:0,Class:20
 2018-12-06 14:03:21,811 [P9144/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-12-06 14:03:22,812 [P9144/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: ARIEL-42\MSSQLSERVER14, InitialCatalog: Roznamah2. Exception: System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:-1,State:0,Class:20
 2018-12-06 14:03:22,812 [P9144/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-12-06 14:03:23,814 [P9144/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: ARIEL-42\MSSQLSERVER14, InitialCatalog: Roznamah2. Exception: System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:-1,State:0,Class:20
 2018-12-06 14:03:23,814 [P9144/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-12-06 14:03:39,350 [P9144/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: ARIEL-42\MSSQLSERVER14, InitialCatalog: Roznamah2. Exception: System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:-1,State:0,Class:20
 2018-12-06 14:03:39,350 [P9144/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2018-12-06 14:03:39,460 [P9144/D2/T1] ERROR Umbraco.Core.UmbracoApplicationBase - An unhandled exception occurred
Umbraco.Core.Exceptions.UmbracoStartupFailedException: Umbraco cannot start. A connection string is configured but Umbraco cannot connect to the database.
   at Umbraco.Core.CoreBootManager.EnsureDatabaseConnection()
   at Umbraco.Core.CoreBootManager.Complete(Action`1 afterComplete)
   at Umbraco.Web.WebBootManager.Complete(Action`1 afterComplete)
   at Umbraco.Core.UmbracoApplicationBase.StartApplication(Object sender, EventArgs e)
 2018-12-06 14:03:49,686 [P9144/D2/T13] INFO  Umbraco.Core.MainDom - Stopping...
 2018-12-06 14:03:49,705 [P9144/D2/T13] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2018-12-06 14:03:49,916 [P9144/D2/T13] INFO  Umbraco.Web.ExamineStartup - Complete (took 211ms)
 2018-12-06 14:03:49,945 [P9144/D2/T13] INFO  Umbraco.Core.MainDom - Released MainDom.
 2018-12-06 14:04:52,731 [P6004/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.12.4 application starting on ARIEL-41
 2018-12-06 14:04:52,793 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2018-12-06 14:04:52,805 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 11ms)
 2018-12-06 14:04:53,003 [P6004/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2018-12-06 14:04:53,006 [P6004/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2018-12-06 14:04:53,011 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:04:53,402 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 391ms)
 2018-12-06 14:04:53,402 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2018-12-06 14:04:53,408 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 5ms)
 2018-12-06 14:04:53,542 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:04:53,543 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2018-12-06 14:04:53,543 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2018-12-06 14:04:53,553 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 9ms)
 2018-12-06 14:04:53,553 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:04:53,554 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:04:53,554 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2018-12-06 14:04:53,569 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 15ms)
 2018-12-06 14:04:53,619 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:04:53,619 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:04:53,619 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2018-12-06 14:04:53,633 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 14ms)
 2018-12-06 14:04:53,634 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:04:53,634 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:04:53,691 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2018-12-06 14:04:53,697 [P6004/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 5ms)
 2018-12-06 14:04:57,010 [P6004/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2018-12-06 14:04:58,073 [P6004/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2018-12-06 14:04:58,360 [P6004/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2018-12-06 14:04:58,534 [P6004/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2018-12-06 14:04:58,646 [P6004/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 6071ms)
 2018-12-06 14:04:58,652 [P6004/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalIndexSet to be unlocked since it was left in a locked state
 2018-12-06 14:04:58,653 [P6004/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index ExternalIndexSet to be unlocked since it was left in a locked state
 2018-12-06 14:04:59,595 [P6004/D2/T10] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:8129/umbraco
 2018-12-06 14:04:59,597 [P6004/D2/T10] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:8129/umbraco (UmbracoModule request)
 2018-12-06 14:05:01,006 [P6004/D2/T10] INFO  umbraco.content - Load Xml from file...
 2018-12-06 14:05:01,035 [P6004/D2/T10] INFO  umbraco.content - Loaded Xml from file.
 2018-12-06 14:05:03,475 [P6004/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:05:03,475 [P6004/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:05:03,476 [P6004/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2018-12-06 14:05:03,480 [P6004/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 4ms)
 2018-12-06 14:05:14,057 [P6004/D2/T10] WARN  Umbraco.Web.UmbracoModule - Status code is 404 yet TrySkipIisCustomErrors is false - IIS will take over.
 2018-12-06 14:06:01,274 [P6004/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:06:01,274 [P6004/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:06:01,275 [P6004/D2/T13] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2018-12-06 14:06:01,397 [P6004/D2/T11] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2018-12-05 14:06:01
 2018-12-06 14:06:01,414 [P6004/D2/T13] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 139ms)
 2018-12-06 14:06:01,500 [P6004/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:06:01,500 [P6004/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:06:01,500 [P6004/D2/T12] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ICacheRefresher
 2018-12-06 14:06:01,519 [P6004/D2/T12] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ICacheRefresher (took 18ms)
 2018-12-06 14:06:10,193 [P6004/D2/T12] INFO  umbraco.content - Save Xml to file...
 2018-12-06 14:06:10,239 [P6004/D2/T12] INFO  umbraco.content - Saved Xml to file.
 2018-12-06 14:10:33,075 [P6004/D2/T39] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:10:33,076 [P6004/D2/T39] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:10:33,076 [P6004/D2/T39] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2018-12-06 14:10:33,103 [P6004/D2/T39] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 27ms)
 2018-12-06 14:10:33,811 [P6004/D2/T39] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:10:33,811 [P6004/D2/T39] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:10:33,811 [P6004/D2/T39] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2018-12-06 14:10:33,817 [P6004/D2/T39] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 5ms)
 2018-12-06 14:11:50,437 [P6004/D2/T29] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username amit.chawla.mca@live.com from IP address ::1
 2018-12-06 14:11:50,439 [P6004/D2/T29] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: amit.chawla.mca@live.com logged in from IP address ::1
 2018-12-06 14:11:52,586 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:11:52,587 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:11:52,587 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ITree
 2018-12-06 14:11:52,598 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ITree (took 11ms)
 2018-12-06 14:11:52,642 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:11:52,642 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:11:52,643 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2018-12-06 14:11:52,643 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2018-12-06 14:11:52,648 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:11:52,648 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:11:52,649 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2018-12-06 14:11:52,654 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 5ms)
 2018-12-06 14:11:52,705 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 14:11:52,706 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 14:11:52,706 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2018-12-06 14:11:52,711 [P6004/D2/T36] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 5ms)
 2018-12-06 14:11:57,185 [P6004/D2/T40] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 14:11:57,269 [P6004/D2/T36] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field multiPicker
 2018-12-06 14:11:57,269 [P6004/D2/T36] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field onlyImages
 2018-12-06 14:11:57,269 [P6004/D2/T36] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field disableFolderSelect
 2018-12-06 14:11:57,270 [P6004/D2/T36] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field startNodeId
 2018-12-06 14:11:59,227 [P6004/D2/T40] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 14:11:59,646 [P6004/D2/T39] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field default
 2018-12-06 14:11:59,872 [P6004/D2/T39] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 14:12:00,016 [P6004/D2/T40] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 14:12:00,125 [P6004/D2/T39] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 14:12:00,392 [P6004/D2/T29] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 14:12:00,397 [P6004/D2/T7] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 14:12:00,398 [P6004/D2/T12] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 14:12:00,423 [P6004/D2/T35] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 14:12:00,536 [P6004/D2/T12] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 14:12:00,546 [P6004/D2/T23] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 14:12:00,697 [P6004/D2/T37] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 14:31:25,969 [P6004/D2/T83] INFO  Umbraco.Core.Logging.OwinLogger - Event Id: 0, state: User logged will be logged out due to timeout: amit.chawla.mca@live.com, IP Address: ::1
 2018-12-06 14:56:43,350 [P6004/D2/T228] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username amit.chawla.mca@live.com from IP address ::1
 2018-12-06 14:56:43,351 [P6004/D2/T228] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: amit.chawla.mca@live.com logged in from IP address ::1
 2018-12-06 14:57:50,995 [P6004/D2/T236] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'AR' with Id '1058' has been published.
 2018-12-06 14:57:58,335 [P6004/D2/T228] INFO  umbraco.content - Save Xml to file...
 2018-12-06 14:57:58,341 [P6004/D2/T228] INFO  umbraco.content - Saved Xml to file.
 2018-12-06 15:01:33,393 [P6004/D2/T212] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'King Fahad Stadium' with Id '2071' has been published.
 2018-12-06 15:01:44,035 [P6004/D2/T200] INFO  umbraco.content - Save Xml to file...
 2018-12-06 15:01:44,039 [P6004/D2/T200] INFO  umbraco.content - Saved Xml to file.
 2018-12-06 15:01:45,257 [P6004/D2/T212] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'King Fahad Stadium' with Id '2071' has been published.
 2018-12-06 15:01:55,438 [P6004/D2/T212] INFO  umbraco.content - Save Xml to file...
 2018-12-06 15:01:55,444 [P6004/D2/T212] INFO  umbraco.content - Saved Xml to file.
 2018-12-06 15:02:08,714 [P6004/D2/T216] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 15:02:08,714 [P6004/D2/T216] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 15:02:08,714 [P6004/D2/T216] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Deploy.IServiceConnector
 2018-12-06 15:02:08,721 [P6004/D2/T216] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Deploy.IServiceConnector (took 7ms)
 2018-12-06 15:02:14,088 [P6004/D2/T239] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Riyadh International Convention & Exhibition Center' with Id '1088' has been published.
 2018-12-06 15:02:23,238 [P6004/D2/T215] INFO  umbraco.content - Save Xml to file...
 2018-12-06 15:02:23,246 [P6004/D2/T215] INFO  umbraco.content - Saved Xml to file.
 2018-12-06 15:05:01,113 [P6004/D2/T194] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 15:05:01,114 [P6004/D2/T194] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 15:05:01,114 [P6004/D2/T194] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Editors.IEditorValidator
 2018-12-06 15:05:01,129 [P6004/D2/T194] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Editors.IEditorValidator (took 15ms)
 2018-12-06 15:05:34,701 [P6004/D2/T194] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field default
 2018-12-06 15:05:34,729 [P6004/D2/T229] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:05:34,731 [P6004/D2/T205] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:05:34,784 [P6004/D2/T194] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:05:34,784 [P6004/D2/T229] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:05:34,784 [P6004/D2/T197] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:05:34,839 [P6004/D2/T205] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:33:28,426 [P6004/D2/T189] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field multiPicker
 2018-12-06 15:33:28,426 [P6004/D2/T189] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field onlyImages
 2018-12-06 15:33:28,427 [P6004/D2/T189] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field disableFolderSelect
 2018-12-06 15:33:28,427 [P6004/D2/T189] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field startNodeId
 2018-12-06 15:33:28,442 [P6004/D2/T192] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:33:38,199 [P6004/D2/T219] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:33:38,677 [P6004/D2/T192] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field default
 2018-12-06 15:33:38,768 [P6004/D2/T220] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:33:39,450 [P6004/D2/T217] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:33:40,324 [P6004/D2/T218] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:33:40,720 [P6004/D2/T220] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:33:41,085 [P6004/D2/T217] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:33:41,094 [P6004/D2/T219] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:33:41,832 [P6004/D2/T160] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:33:45,032 [P6004/D2/T192] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:33:45,036 [P6004/D2/T213] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:33:46,317 [P6004/D2/T189] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:36:07,162 [P6004/D2/T219] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:36:07,438 [P6004/D2/T192] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field default
 2018-12-06 15:36:07,454 [P6004/D2/T189] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:36:08,768 [P6004/D2/T237] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:36:09,149 [P6004/D2/T233] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field tabName
 2018-12-06 15:36:09,820 [P6004/D2/T192] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:36:09,826 [P6004/D2/T219] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:36:09,826 [P6004/D2/T208] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:36:11,774 [P6004/D2/T189] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field tabName
 2018-12-06 15:39:13,253 [P6004/D2/T157] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field tabName
 2018-12-06 15:39:23,540 [P6004/D2/T230] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:39:23,552 [P6004/D2/T225] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:39:24,449 [P6004/D2/T207] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field default
 2018-12-06 15:39:26,075 [P6004/D2/T232] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:39:26,365 [P6004/D2/T207] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:39:26,658 [P6004/D2/T225] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field tabName
 2018-12-06 15:39:27,624 [P6004/D2/T217] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:39:27,833 [P6004/D2/T207] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 15:39:30,101 [P6004/D2/T157] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field tabName
 2018-12-06 15:41:31,430 [P6004/D2/T186] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field tabName
 2018-12-06 15:47:39,647 [P6004/D2/T189] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'King Fahad Stadium' with Id '2071' has been published.
 2018-12-06 15:47:48,622 [P6004/D2/T189] INFO  umbraco.content - Save Xml to file...
 2018-12-06 15:47:48,626 [P6004/D2/T189] INFO  umbraco.content - Saved Xml to file.
 2018-12-06 15:48:19,503 [P6004/D2/T203] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Riyadh International Convention & Exhibition Center' with Id '1088' has been published.
 2018-12-06 15:48:29,051 [P6004/D2/T203] INFO  umbraco.content - Save Xml to file...
 2018-12-06 15:48:29,055 [P6004/D2/T203] INFO  umbraco.content - Saved Xml to file.
 2018-12-06 15:56:47,410 [P6004/D2/T190] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Zwart 180 Degrees' with Id '4135' has been published.
 2018-12-06 15:56:59,299 [P6004/D2/T190] INFO  umbraco.content - Save Xml to file...
 2018-12-06 15:56:59,304 [P6004/D2/T190] INFO  umbraco.content - Saved Xml to file.
 2018-12-06 16:15:43,980 [P6004/D2/T212] INFO  Umbraco.Core.Logging.OwinLogger - Event Id: 0, state: User logged will be logged out due to timeout: amit.chawla.mca@live.com, IP Address: ::1
 2018-12-06 16:15:44,967 [P6004/D2/T184] INFO  Umbraco.Core.Logging.OwinLogger - Event Id: 0, state: User logged will be logged out due to timeout: amit.chawla.mca@live.com, IP Address: ::1
 2018-12-06 16:35:43,378 [P9552/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.12.4 application starting on ARIEL-41
 2018-12-06 16:35:43,412 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2018-12-06 16:35:43,421 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 8ms)
 2018-12-06 16:35:43,524 [P9552/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2018-12-06 16:35:43,526 [P9552/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2018-12-06 16:35:43,529 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:35:44,295 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 766ms)
 2018-12-06 16:35:44,295 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2018-12-06 16:35:44,297 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 1ms)
 2018-12-06 16:35:44,489 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:35:44,490 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2018-12-06 16:35:44,490 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2018-12-06 16:35:44,490 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 0ms)
 2018-12-06 16:35:44,490 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:35:44,491 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:35:44,491 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2018-12-06 16:35:44,493 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 1ms)
 2018-12-06 16:35:44,512 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:35:44,512 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:35:44,513 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2018-12-06 16:35:44,513 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 0ms)
 2018-12-06 16:35:44,513 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:35:44,513 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:35:44,513 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2018-12-06 16:35:44,514 [P9552/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 0ms)
 2018-12-06 16:35:47,467 [P9552/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2018-12-06 16:35:48,117 [P9552/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2018-12-06 16:35:48,532 [P9552/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2018-12-06 16:35:48,695 [P9552/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2018-12-06 16:35:48,712 [P9552/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 5633ms)
 2018-12-06 16:35:48,717 [P9552/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index InternalIndexSet to be unlocked since it was left in a locked state
 2018-12-06 16:35:48,718 [P9552/D2/T1] INFO  Umbraco.Web.ExamineStartup - Forcing index ExternalIndexSet to be unlocked since it was left in a locked state
 2018-12-06 16:35:51,960 [P9552/D2/T9] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://localhost:8129/umbraco
 2018-12-06 16:35:51,962 [P9552/D2/T9] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:8129/umbraco (UmbracoModule request)
 2018-12-06 16:35:52,807 [P9552/D2/T9] INFO  umbraco.content - Load Xml from file...
 2018-12-06 16:35:52,819 [P9552/D2/T9] INFO  umbraco.content - Loaded Xml from file.
 2018-12-06 16:35:54,093 [P9552/D2/T9] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:35:54,093 [P9552/D2/T9] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:35:54,094 [P9552/D2/T9] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2018-12-06 16:35:54,095 [P9552/D2/T9] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 0ms)
 2018-12-06 16:35:55,916 [P9552/D2/T9] WARN  Umbraco.Web.UmbracoModule - Status code is 404 yet TrySkipIisCustomErrors is false - IIS will take over.
 2018-12-06 16:36:53,015 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:36:53,015 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:36:53,015 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2018-12-06 16:36:53,016 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 0ms)
 2018-12-06 16:36:53,018 [P9552/D2/T5] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2018-12-05 16:36:52
 2018-12-06 16:38:15,546 [P9552/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:38:15,546 [P9552/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:38:15,546 [P9552/D2/T15] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2018-12-06 16:38:15,546 [P9552/D2/T15] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2018-12-06 16:38:15,763 [P9552/D2/T5] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:38:15,764 [P9552/D2/T5] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:38:15,764 [P9552/D2/T5] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IAction
 2018-12-06 16:38:15,765 [P9552/D2/T5] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IAction (took 0ms)
 2018-12-06 16:38:26,478 [P9552/D2/T15] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username amit.chawla.mca@live.com from IP address ::1
 2018-12-06 16:38:26,480 [P9552/D2/T15] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: amit.chawla.mca@live.com logged in from IP address ::1
 2018-12-06 16:38:27,891 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:38:27,891 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:38:27,891 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ITree
 2018-12-06 16:38:27,891 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ITree (took 0ms)
 2018-12-06 16:38:27,907 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:38:27,907 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:38:27,907 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController
 2018-12-06 16:38:27,907 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Web.Trees.TreeAttribute]Umbraco.Web.Trees.TreeController (took 0ms)
 2018-12-06 16:38:27,923 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:38:27,923 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:38:27,923 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree
 2018-12-06 16:38:27,923 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.TreeAttribute]umbraco.interfaces.ITree (took 0ms)
 2018-12-06 16:38:27,938 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:38:27,938 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:38:27,938 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolving [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication
 2018-12-06 16:38:27,938 [P9552/D2/T10] INFO  Umbraco.Core.PluginManager - Resolved [umbraco.businesslogic.ApplicationAttribute]umbraco.interfaces.IApplication (took 0ms)
 2018-12-06 16:38:30,576 [P9552/D2/T10] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 16:38:30,919 [P9552/D2/T22] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field multiPicker
 2018-12-06 16:38:30,919 [P9552/D2/T22] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field onlyImages
 2018-12-06 16:38:30,919 [P9552/D2/T22] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field disableFolderSelect
 2018-12-06 16:38:30,919 [P9552/D2/T22] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field startNodeId
 2018-12-06 16:38:32,566 [P9552/D2/T22] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 16:38:32,690 [P9552/D2/T10] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field default
 2018-12-06 16:38:32,960 [P9552/D2/T10] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 16:38:33,021 [P9552/D2/T10] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 16:38:33,052 [P9552/D2/T10] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 16:38:33,073 [P9552/D2/T10] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 16:38:33,091 [P9552/D2/T10] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 16:38:33,130 [P9552/D2/T10] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 16:38:33,155 [P9552/D2/T15] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 16:38:33,238 [P9552/D2/T22] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 16:38:33,264 [P9552/D2/T20] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 16:38:33,291 [P9552/D2/T16] WARN  Umbraco.Web.Models.Mapping.PreValueDisplayResolver - Could not find persisted pre-value for field maxChars
 2018-12-06 16:41:05,909 [P9552/D2/T32] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:41:05,909 [P9552/D2/T32] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:41:05,910 [P9552/D2/T32] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.Deploy.IServiceConnector
 2018-12-06 16:41:05,911 [P9552/D2/T32] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.Deploy.IServiceConnector (took 1ms)
 2018-12-06 16:41:39,840 [P9552/D2/T28] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Zwart 180 Degrees' with Id '4135' has been published.
 2018-12-06 16:41:44,573 [P9552/D2/T28] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2018-12-06 16:41:44,574 [P9552/D2/T28] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2018-12-06 16:41:44,574 [P9552/D2/T28] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.ICacheRefresher
 2018-12-06 16:41:44,574 [P9552/D2/T28] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.ICacheRefresher (took 0ms)
 2018-12-06 16:41:49,379 [P9552/D2/T28] INFO  umbraco.content - Save Xml to file...
 2018-12-06 16:41:49,390 [P9552/D2/T28] INFO  umbraco.content - Saved Xml to file.
 2018-12-06 16:41:55,515 [P9552/D2/T28] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Zwart 180 Degrees' with Id '4135' will not be published, the event was cancelled.
 2018-12-06 16:42:37,563 [P9552/D2/T36] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Zwart 180 Degrees' with Id '4135' has been published.
 2018-12-06 16:42:46,739 [P9552/D2/T37] INFO  umbraco.content - Save Xml to file...
 2018-12-06 16:42:46,745 [P9552/D2/T37] INFO  umbraco.content - Saved Xml to file.
 2018-12-06 16:49:39,414 [P9552/D2/T14] WARN  Umbraco.Web.UmbracoModule - Status code is 404 yet TrySkipIisCustomErrors is false - IIS will take over.
 2018-12-06 16:51:30,793 [P9552/D2/T51] INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'Abha' with Id '0' has been published.
 2018-12-06 16:51:37,186 [P9552/D2/T57] INFO  umbraco.content - Save Xml to file...
 2018-12-06 16:51:37,195 [P9552/D2/T57] INFO  umbraco.content - Saved Xml to file.
